test/test-helper.js

```diff
@@ -0,0 +1,12 @@
+var Game = require('../lib/game');
+var Board = require('../lib/board');
+var Loader = require('../lib/loader');
+
+beforeEach (function () {
```

Comment:
Cool! @stevekinney, do you have any opinions on doing this set up?

IMO:
The con is that you're seeing up these variables for every test, even if you don't use them, so it could slow things down but given how small the project is, it's not a big concern.